composer.json

```
@@ -20,7 +20,7 @@
   "require-dev": {
     "phpstan/phpstan": "^1.10.5",
     "phpstan/phpstan-symfony": "^1.2.20",
-    "gotenberg/gotenberg-php": "^1.0.0",
+    "gotenberg/gotenberg-php": "^1.0||^2.0",
```

```suggestion
    "gotenberg/gotenberg-php": "^1.0 || ^2.0",
```
just some spacing to keep consistent with the "style"

---

Could you also add this to the `composer suggest` line, as this change is for the `require-dev`